requirements.txt

```diff
@@ -1 +1,5 @@
-
+beautifulsoup4==4.12.0
+mypy==1.9.0
```

Comment:
requirements should include only those libraries that fuel work of your lab. test and check related things should not be present here

Reply:
removed it